### PR TITLE
Update page template to mention cookie banner

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Also corrected typo ('contain' -> 'container').
Please do not merge till we've released cookie banner.

### CODE REVIEW

File: src/styles/page-template/index.md.njk [hunk 1]
```diff
@@ -132,12 +132,14 @@ To change the components that are included in the page template by default, set
...
     </tr>
-    
+
     <tr class="govuk-table__row">
       <td class="govuk-table__cell">bodyStart</td>
       <td class="govuk-table__cell">Block</td>
       <td class="govuk-table__cell">
         Add content after the opening <code>&lt;body&gt;</code> element.
+        <br>
+        For example: The cookie banner component.
```
Comment (EoinShaughnessy) [inline src/styles/page-template/index.md.njk:142]: We need the URL to link from the words 'cookie banner' to the guidance.

Comment (vanitabarrett) [reply src/styles/page-template/index.md.njk:142]: @EoinShaughnessy Linking to `/components/cookie-banner/` should work, as that's what the URL will be once it's published 🙂

Comment (EoinShaughnessy) [inline src/styles/page-template/index.md.njk:142]:
```suggestion
        For example: The [cookie banner](https://design-system.service.gov.uk/components/cookie banner/)component.
```

Comment (EoinShaughnessy) [reply src/styles/page-template/index.md.njk:142]: @vanitabarrett Or should this be:

For example: The [cookie banner](../../components/cookie-banner/).

Comment (vanitabarrett) [reply src/styles/page-template/index.md.njk:142]: @EoinShaughnessy We can write it like this:

```
[cookie banner](/components/cookie-banner/) component.
```

Comment (EoinShaughnessy) [reply src/styles/page-template/index.md.njk:142]: Thanks, @vanitabarrett!

**Review by 36degrees — APPROVED** [review]

**Pull request merged**